lifecycle-image/Dockerfile

```
@@ -0,0 +1,27 @@
+# Stage 1
+FROM ubuntu:bionic AS download-and-extract-lifecycle
+
+ARG LIFECYCLE_VERSION
+ARG LIFECYCLE_GITHUB_ORG
+ARG LIFECYCLE_TGZ=lifecycle-v${LIFECYCLE_VERSION}+linux.x86-64.tgz
+ARG LIFECYCLE_URI="https://github.com/${LIFECYCLE_GITHUB_ORG}/lifecycle/releases/download/v${LIFECYCLE_VERSION}/${LIFECYCLE_TGZ}"
+
+WORKDIR /tmp
+
+RUN apt-get update && apt-get install -y wget
+RUN wget $LIFECYCLE_URI && tar xzvf $LIFECYCLE_TGZ
```

```suggestion
RUN apt-get update && apt-get install -y wget -O lifecycle.tgz
RUN wget $LIFECYCLE_URI && tar xzvf lifecycle.tgz
```

If we name the output document explicitly we won't have to correctly derive the name of the file at `LIFECYCLE_URI`. This will make it easier use a build arg to change the lifecycle uri correctly.